content/en/docs/setup/release/notes.md

```diff
@@ -63,11 +63,9 @@ filename | sha512 hash
 ## Changelog since v1.17.0
 
 A complete changelog for the release notes is now hosted in a customizable
-format at [https://relnotes.k8s.io][1]. Check it out and please give us your
+format at [https://relnotes.k8s.io](https://relnotes.k8s.io/?releaseVersions=1.18.0). Check it out and please give us your
```

Comment:
nit: This is OK. Though, the change needs to be made, at some point, in the release notes generator?

Reply:
Seems like an issue that needs filing as a follow-up?

Reply:
Alternative: teach the link checker about both kinds of Markdown hyperlink?

Reply:
Thought about that but .... it means we need to scan the same markdown twice: the first scan identifies link targets like `[bar]: https://somewhere.com`, and the second scan validates all links and see if they hit `[foo][bar]`. I didn't do that because the occurrences is manageable so it doesn't worth the effort.